omod/src/main/webapp/resources/htmlforms/HMIS-ACP-029-SafeMaleCircumcisionClientForm.xml

```diff
@@ -11,1078 +11,597 @@
-                                                            <i class=" icon-calendar small family-date-icon position-absolute" style="right: 25%; top: 50%;" ></i>
-                                                             <obs conceptId="165390" class="hasDatepicker" allowFutureDates="false" />
-                                                        </div>
-                                                    </div>
-                                                    <div class="col-md-4">
-                                                        <label>Time started:</label>
-                                                        <span class="time-box-horizontal">
-                                                            <obs conceptId="99542" id="99542"/>
-                                                        </span>
-                                                    </div>
-                                                    <div class="col-md-4">
-                                                        <label>End Time:</label>
-                                                        <span class="time-box-horizontal">
-                                                            <obs conceptId="99543" id="99543"/>
-                                                        </span>
-
-                                                    </div>
-                                                </div>
-
-                                                <!--  Anasthesia   -->
-                                                <div class="row">
-                                                    <div class="col-md-3"><label>Local Anesthesia</label> </div>
-                                                    <div class="col-md-4 ">
-                                                        <label>Lignocaine (mls)</label>
-                                                        <obs conceptId="99671" />
-                                                    </div>
-                                                    <div class="col-md-3">
-                                                        <label>Bupivicaine (mls)</label>
-                                                        <obs conceptId="99672" />
-                                                    </div>
-                                                    <div class="col-md-2 col-md-offset-2">
-                                                        <obs conceptId="99612" answerConceptId="165384" answerLabel="EMLA Cream" style="checkbox"/>
-                                                    </div>
-                                                </div>
-
-                                                <!--  Type of circumcision procedure  -->
-                                                <div class="row ">
-                                                    <div class="col-md-3">
-                                                        <label>Type of Circumcision Proc
```

Comment:
Insert a medication set, so that its auto complete

Reply:
Done